src/themes/teams/components/Button/buttonVariables.ts

```diff
@@ -25,6 +26,7 @@ export default (siteVars: any): IButtonVariables => {
     height: pxToRem(32),
     minWidth: pxToRem(96),
     maxWidth: pxToRem(280),
+    color: undefined,
```

Comment:
the motivation for this variable is to allow to customize Button's color (where 'color' comes in general sense, not something that is tied to corresponding CSS property only). This will open way for the following customization scenarios, as well as allow to customize this aspect of `Button` component in docs

```jsx
<Button variables={{ color: 'red' }} ... />
```

> Seems strange to have an undefined variable that is also never set to a value. Is there not a siteVariable available for this value?

Yes, let me initialise it with an appropriate site variable - the reason haven't done it before is because it was unclear to me at this moment which ones will be provided by teams theme